packages/@aws-cdk/aws-stepfunctions-tasks/test/emrcontainers/integ.start-job-run.ts

```diff
@@ -16,8 +16,24 @@ import { Classification, ReleaseLabel } from '../../lib/emrcontainers/start-job-
 const app = new cdk.App();
 const stack = new cdk.Stack(app, 'aws-stepfunctions-tasks-emr-containers-start-job-run-integ-test');
 
-const startJobRunJob = new EmrContainersStartJobRun(stack, 'EMR Containers Start Job Run', {
-  virtualClusterId: sfn.TaskInput.fromText('frv3my91yzjr1g0ha2h91obx3'),
+const createVirtualCluster = new cdk.CfnResource(stack, 'Create-a-virtual-cluster', {
+  type: 'AWS::EMRContainers::VirtualCluster',
+  properties: {
+    ContainerProvider: {
+      Id: 'test-eks',
```

Comment:
you need to create an EKS cluster in this app and pass its ID here

Reply:
Again the whole enabling [cluster access ](https://docs.aws.amazon.com/emr/latest/EMR-on-EKS-DevelopmentGuide/setting-up-cluster-access.html) and [IAM role ](https://docs.aws.amazon.com/emr/latest/EMR-on-EKS-DevelopmentGuide/setting-up-enable-IAM.html) problem.

Reply:
Scrapping the integration tests seem like a better idea at this point. I see some other packages that don't have integration tests either. Like the ``emr` package

Reply:
This PR will not be merged without integration tests. There are too many moving parts that we cannot confirm to be correct without an actual deplorable app